Containerfile

```diff
@@ -1,7 +1,7 @@
 FROM mambaorg/micromamba:1.5.8
 
 LABEL maintainer="Microbiome Informatics"
-LABEL version="1.0.0"
+LABEL version="1.0.1"
```

Comment:
Please also bump the version in this file -> https://github.com/EBI-Metagenomics/fetch_tool/blob/develop/fetchtool/__init__.py

Reply:
Addressed